### DISCUSSION TITLE
Updates for WP 6.6.1

### DISCUSSION THREAD

**Discussion by mteedev** [GuduleLapointe/w4os#78] [General]
Hello, i was wondering if there is an update in the works for the WordPress v6.6.1 - when I try to activate this it lags and causes the whole site to slow way down. Thanks!!!

**Comment by GuduleLapointe** [comment]
There should be no difference between the displayed WP version and 6.6.1 in the way the plugin works, so no update is planned solely for that, the last compatible WP version will be updated with the next other updates.

I tried myself to benchmark the website speed while activating and deactivating the plugin, I get no noticeable difference.

That said, it could be related to issue #77 . Could you check your web log files for things like url not found?

A work is in progress to stop querying misconfigured URLs, which could be a cause of slowdown.

**Reply by mteedev** [reply]
I installed this latest version and have the slow-to-activate issue and after activation, the whole Wordpress is slow. Here are error logs:


1 | PHP Deprecated | Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameterPlugin: w4os - OpenSimulator Web InterfaceFile: /wp-content/plugins/w4os-opensimulator-web-interface/blocks/web-search.phpLine: 342 | Aug 12, 2024 - 03:00:51(1 occurrence logged)
-- | -- | -- | --
2 | PHP Deprecated | Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameterPlugin: w4os - OpenSimulator Web InterfaceFile: /wp-content/plugins/w4os-opensimulator-web-interface/blocks/popular-places.phpLine: 318 | Aug 12, 2024 - 03:00:51(1 occurrence logged)
3 | PHP Deprecated | Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameterPlugin: w4os - OpenSimulator Web InterfaceFile: /wp-content/plugins/w4os-opensimulator-web-interface/blocks/avatar-profile.phpLine: 265 | Aug 12, 2024 - 03:00:51(1 occurrence logged)
4 | PHP Deprecated | Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameterPlugin: w4os - OpenSimulator Web InterfaceFile: /wp-content/plugins/w4os-opensimulator-web-interface/blocks/grid-status.phpLine: 202 | Aug 12, 2024 - 03:00:51(1 occurrence logged)
5 | PHP Deprecated | Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameterPlugin: w4os - OpenSimulator Web InterfaceFile: /wp-content/plugins/w4os-opensimulator-web-interface/blocks/grid-info.phpLine: 250 | Aug 12, 2024 - 03:00:51(1 occurrence logged)
6 | Other | Cron reschedule event error for hook: action_scheduler_run_queue, Error code: invalid_schedule, Error message: Event schedule does not exist., Data: {"schedule":"every_minute","args":["WP Cron"],"interval":60} | Aug 12, 2024 - 02:59:22(1 occurrence logged)


</body>
</html>1	PHP Deprecated	Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameter
Plugin: [w4os - OpenSimulator Web Interface](https://w4os.org/)
File: [/wp-content/plugins/w4os-opensimulator-web-interface/blocks/web-search.php](https://nvwdev.masalaestates.net/wp-admin/plugin-editor.php?file=w4os-opensimulator-web-interface%2Fblocks%2Fweb-search.php&plugin=w4os-opensimulator-web-interface%2Fw4os.php)
Line: 342	Aug 12, 2024 - 03:00:51
(1 occurrence logged)
2	PHP Deprecated	Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameter
Plugin: [w4os - OpenSimulator Web Interface](https://w4os.org/)
File: [/wp-content/plugins/w4os-opensimulator-web-interface/blocks/popular-places.php](https://nvwdev.masalaestates.net/wp-admin/plugin-editor.php?file=w4os-opensimulator-web-interface%2Fblocks%2Fpopular-places.php&plugin=w4os-opensimulator-web-interface%2Fw4os.php)
Line: 318	Aug 12, 2024 - 03:00:51
(1 occurrence logged)
3	PHP Deprecated	Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameter
Plugin: [w4os - OpenSimulator Web Interface](https://w4os.org/)
File: [/wp-content/plugins/w4os-opensimulator-web-interface/blocks/avatar-profile.php](https://nvwdev.masalaestates.net/wp-admin/plugin-editor.php?file=w4os-opensimulator-web-interface%2Fblocks%2Favatar-profile.php&plugin=w4os-opensimulator-web-interface%2Fw4os.php)
Line: 265	Aug 12, 2024 - 03:00:51
(1 occurrence logged)
4	PHP Deprecated	Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameter
Plugin: [w4os - OpenSimulator Web Interface](https://w4os.org/)
File: [/wp-content/plugins/w4os-opensimulator-web-interface/blocks/grid-status.php](https://nvwdev.masalaestates.net/wp-admin/plugin-editor.php?file=w4os-opensimulator-web-interface%2Fblocks%2Fgrid-status.php&plugin=w4os-opensimulator-web-interface%2Fw4os.php)
Line: 202	Aug 12, 2024 - 03:00:51
(1 occurrence logged)
5	PHP Deprecated	Optional parameter $content declared before required parameter $function_name is implicitly treated as a required parameter
Plugin: [w4os - OpenSimulator Web Interface](https://w4os.org/)
File: [/wp-content/plugins/w4os-opensimulator-web-interface/blocks/grid-info.php](https://nvwdev.masalaestates.net/wp-admin/plugin-editor.php?file=w4os-opensimulator-web-interface%2Fblocks%2Fgrid-info.php&plugin=w4os-opensimulator-web-interface%2Fw4os.php)
Line: 250	Aug 12, 2024 - 03:00:51
(1 occurrence logged)
6	Other	Cron reschedule event error for hook: action_scheduler_run_queue, Error code: invalid_schedule, Error message: Event schedule does not exist., Data: {"schedule":"every_minute","args":["WP Cron"],"interval":60}	Aug 12, 2024 - 02:59:22
(1 occurrence logged)

**Comment by GuduleLapointe** [comment]
Tested version fixed in 2.7.7 ff4191a80e8b1d74e2dd55ac0b3768cb7ef1ef6b

**Comment by GuduleLapointe** [comment]
I created a separate issue for this, as it is another problem than the solved WP version, please follow #80 from now on, I close this discussion.
